mne/report.py

```diff
@@ -1052,6 +1052,22 @@ def add_custom_css(self, css):
         style = f'\n<style type="text/css">\n{css}\n</style>'
         self.include += style
 
+    def add_custom_js(self, js):
+        """Add custom JavaScript to the report.
+
+        Parameters
+        ----------
+        js : str
+            JavaScript code to add to the report. The content of this string
+            will be embedded between HTML ``<script>`` and ``</script>`` tags.
+
+        Notes
+        -----
+        .. versionadded:: 0.23
+        """
+        script = f'\n<script type="text/javascript">\n{js}\n</script>'
```

Comment:
I can't quite pinpoint the issue but this feels a bit hacky. I guess it might still work but [see issues like this](https://stackoverflow.com/questions/3037725/is-it-wrong-to-place-the-script-tag-after-the-body-tag). Ideally you'd want the `report.html` to be some sort of `OrderedDict` instead of list so that you can access the existing <script> elements and append it there rather than make a new element in the list.

Not pushing for it, but just a mild suggestion if someone is bored on a rainy afternoon :)

Reply:
I like your suggestion, care to open an issue as a reminder? 🙂